mantidproject/mantid
# Convert Mac line endings to Unix for CHESS

**Description of work.**

Mac-only eol characters confuse our checksum calculator
and it causes the IDF to be downloaded even when this is not
necessary and the downloaded content matches the installed
content.

**To test:**

To check the issue before the merge:
* Close Workbench
* Remove `github.json` & all `.xml` files in `~/.mantid/instrument` (Mac/Linux) or `%APPDATA%\mantidproject\mantid\instrument`
* Add `UpdateInstrumentDefinitions.OnStartup = 1` to `Mantid.user.properties`
* Start Workbench and observe that 1 new definition is downloaded.
* Check `~/.mantid/instrument` or `%APPDATA%\mantidproject\mantid\instrument` and you should see `CHESS_Definition.xml`
* Close Workbench
* Remove `github.json` & all `.xml` files in `~/.mantid/instrument` (Mac/Linux) or `%APPDATA%\mantidproject\mantid\instrument` again.

Merge the fix and redo the above steps but now nothing should be reported as downloaded.

*There is no associated issue.*

*This does not require release notes* because **it's a new IDF for this cycle.**


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
